celery/beat.py

```diff
@@ -281,6 +284,18 @@ def tick(self, event_t=event_t, min=min, heappop=heapq.heappop,
                 return min(verify[0], max_interval)
         return min(adjust(next_time_to_run) or max_interval, max_interval)
 
+    def schedules_equal(self, a, b):
+        if a.keys() != b.keys():
+            return False
+        for name, model in a.items():
+            b_model = b.get(name)
+            if not b_model:
+                return False
+            if (hasattr(model.schedule, '__repr__') and
+                    model.schedule.__repr__() != b_model.schedule.__repr__()):
```

Comment:
Oh, wait, my objection to my own pull request might be unfounded.

`django_celery_beat.schedulers.ModelEntry` is actually inherited from `celery.beat ScheduleEntry` - so I am guessing is actually a standard celery object with some extra bits added.